AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/StateMachine/CodeGen/Data/PasswordProtectionSettings.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import Amplify
+
+@_spi(InternalAmplifyConfiguration)
+public struct PasswordProtectionSettings: Equatable, Codable {
+    public var minLength: Int?
```

Comment:
I'd rather have this as a non-optional `UInt` that just defaults to `0`.

Also, can the properties be `let` instead?

Reply:
updated